WeakAuras/GenericTrigger.lua

```diff
@@ -3665,6 +3665,19 @@ function GenericTrigger.GetAdditionalProperties(data, triggernum)
         ret = ret .. additional;
       end
     end
+  else
+    if (trigger.custom_type == "stateupdate") then
+      local variables = events[data.id][triggernum].tsuConditionVariables();
+      if (type(variables) == "table") then
+        for var, varData in pairs(variables) do
+          if (type(varData) == "table") then
+            if varData.description then
```

Comment:
Are you sure you want to add a new field ? I though it was going to use the previously supported  `display` field